<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roulette</class>
 <widget class="QWidget" name="roulette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Stop">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>750</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>멈춰!</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>570</y>
     <width>500</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Retry_Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>670</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>다시 할래!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Get_Out_Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>670</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>나갈래</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
